packages/editor/src/utils/typesafe-telemetry-actions.ts

```diff
@@ -0,0 +1,52 @@
+import { createAction as defaultCreateAction } from 'typesafe-actions';
+
+interface ITelemetryMetadata {
+  eventName: string;
+}
+
+interface IMetadata {
+  telemetry?: ITelemetryMetadata;
+}
+
+interface ICreateActionOptions {
+  shouldSendTelemetry?: boolean;
+  addTimestamp?: boolean;
+}
+
+const defaultOptions = { shouldSendTelemetry: false, addTimestamp: false };
```

Comment:
We can't be collecting telmetry until we go through a privacy review.  Could you add a constant here, like
```
// For now, until complete privacy review, only enabling telemetry in local dev environment, not production:
const TELEMETRY_ENABLED = process.env.NODE_ENV !== 'production'
```

... and then make use of it below?

Reply:
@jakobpn

Reply:
Fine, I'll do it when it gets merged in.